utils/src/main/kotlin/Utils.kt

```diff
@@ -119,13 +119,13 @@ fun filterVersionNames(version: String, names: List<String>, project: String? =
                     val currentSeparators = if (versionHasSeparator) versionVariant.separators else versionSeparators
 
                     // Full match with the current version variant.
-                    last == null
+                    last == null ||
                             // The prefix does not end with the current separators or a digit.
-                            || (last !in currentSeparators && !last.isDigit())
+                            (last !in currentSeparators && !last.isDigit()) ||
                             // The prefix ends with the current separators but the forelast character is not a digit.
-                            || (last in currentSeparators && (forelast == null || !forelast.isDigit()))
+                            (last in currentSeparators && (forelast == null || !forelast.isDigit())) ||
                             // The prefix ends with 'v' and the forelast character is a separator.
-                            || (last == 'v' && (forelast == null || forelast in currentSeparators))
+                            (last == 'v' && (forelast == null || forelast in currentSeparators))
```

Comment:
I prefer line breaks before the operator, not after, and this is a good example where readability is better with the break before the operator. Is it possible to configure detekt to change that rule?

Reply:
There is only a single rule for both occurrences like this and also the cases where long strings are concatenated with `+`. IIUYC, in case of strings you prefer the `+` at the end of the line, but in case of logical operators you prefer them at the start of a line?

Reply:
No, always at the front.

Reply:
Then I'll simply drop the commit.